habitat/core/simulator.py

```diff
@@ -6,27 +6,16 @@
 import abc
 from collections import OrderedDict
 from enum import Enum
-from typing import (
-    TYPE_CHECKING,
-    Any,
-    Dict,
-    Iterable,
-    List,
-    Optional,
-    Sequence,
-    Union,
-)
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Union
 
 import attr
+import numpy as np
+import torch
```

Comment:
@Skylion007, this is the only line dependent on PyTorch in Hab-Lab core. Is there a way we can guard it under TYPE_CHECKING?

Reply:
Yes.